kawin/KWNBase.py

```diff
@@ -228,9 +228,9 @@ def save(self, filename, compressed = False, toCSV = False):
                 csv.writer(f).writerows(rows)
         else:
             if compressed:
-                np.savez_compressed(filename, **vDict)
+                np.savez_compressed(filename, **vDict, allow_pickle=True)
```

Comment:
Why is `allow_pickle` required here? Is it because of the string headers?

Reply:
This was due to a feature I added for custom outputs of the precipitation model where a user can input a function and a name to store the function as in the model. These were kept as dictionaries of {'name': user-specified name, 'function': function object}. This required allow_pickle to be True for saving and loading to work.

I changed this by storing the functions and the given names as separate lists. When saving, the functions will be removed and only the names for the functions and the outputs of the functions will be saved. This removes the need to pickle since everything can be stored as using numpy variable types.